src/EnergyPlus/ChillerReformulatedEIR.cc

```diff
@@ -377,7 +352,7 @@ namespace ChillerReformulatedEIR {
                                                DataIPShortCuts::cAlphaArgs(7),
                                                "Chilled Water Nodes");
 
-            ElecReformEIRChiller(EIRChillerNum).CondenserType = WaterCooled;
+            thisChiller.CondenserType = DataPlant::CondenserType::WATERCOOLED;
```

Comment:
```CondenserType``` and ```FlowMode``` ints were duplicated across multiple components (and there may still be others out there doing the same). These were moved to an ```enum``` in ```DataPlant```.